examples/snippets/tags/application.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Google LLC
+#
+# Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+require "io/console"
+require_relative "../config/environment"
+require_relative "models/singer"
+require_relative "models/album"
+
+class Application
+  def self.run
+    # Query all albums and include a request tag for the statement.
+    puts ""
+    puts "Query all Albums and include a request tag"
+    albums = Album.annotate("request_tag: query-all-albums").all
+    puts "Queried #{albums.length} albums using a request tag"
+
+    # Query all albums in a transaction and include both a request tag and a transaction tag.
```

Comment:
Nits: comment seems not necessary as there is a print-out.

Reply:
Removed